recipes/qh3/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "qh3" %}
+{% set version = "0.14.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jawah/qh3/archive/v{{ version }}.tar.gz
```

Comment:
hello there,

try this instead `https://github.com/jawah/qh3/releases/download/v{{ version }}/qh3-{{ version }}.tar.gz`
will solve the lsqpack missing header.

regards,

Reply:
with `{% set version = "0.15.1" %}` as the starting pts.